### PR TITLE
Part1

### PR DESCRIPTION
**Pull request by acanavesialtimetrik**
Part 1 and 2 finished

### CODE REVIEW

File: src/classes/ExpensesController.cls [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+        if(employee != null){
+          condition += ' AND Employee__r.Id = :employeeId ';
+        }
+        if(condition.length() > 0 ){
+          condition = condition.replaceFirst('AND', ' WHERE ');
+        }
+        String query = String.escapeSingleQuotes(' SELECT  Employee__c, Category__c, Type__c, Amount__c, Tax__c, Client_Company__c, Submitted__c' +
+        ' FROM Expense__c ' +
+        condition+
+        ' LIMIT 100');
```
Comment (rjulianialtimetrik) [inline src/classes/ExpensesController.cls]: Limiting this query to only 100 results might not be ideal. please review Salesforce's Governor Limits and use a number  that makes more sense

File: src/triggers/DeletionRules.trigger [hunk 2]
```diff
@@ -0,0 +1,18 @@
+trigger DeletionRules on Expense__c (after update, after delete) {
```
Comment (rjulianialtimetrik) [inline src/triggers/DeletionRules.trigger:1]: It's recommended to use the object's name as part of a trigger's name. For example, this trigger could be called ExpenseDeletionRules

**Review by rjulianialtimetrik — CHANGES_REQUESTED** [review]

**Review by rjulianialtimetrik — APPROVED** [review]
Looks pretty good! I'm just going to ask you to please add pagination to your list

**Comment by acanavesialtimetrik** [comment]
Yes, pagination is still a pending task, I was finishing other issues related to part 3 and then I get back to here.

**Pull request merged**